go/service/tlf.go

```diff
@@ -60,14 +60,21 @@ func (h *tlfHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, arg key
 	// Append username in case it's not present. We don't need to check if it
 	// exists already since CryptKeys calls below transforms the TLF name into a
 	// canonical one.
-	arg.TlfName = arg.TlfName + "," + string(username)
+	arg.Query.TlfName = arg.Query.TlfName + "," + string(username)
```

Comment:
Yeah; the service is not doing the auto-completion; the CLI code is calling the RPC.

For your example of `keybase chat read max`, do you think we should factor this into the conversation resolver, so that if both `chris,max` and `max#chris` exist, a menu should be presented for user to choose?